menu.js

```diff
@@ -124,3 +124,44 @@ closePopup.addEventListener('click', () => {
   popupContainer.style.display = 'none';
 });
 // End popup window
+
+// Form validation JS
+
+const contactForm = document.getElementById('contact-form');
+const email = document.getElementById('email');
+const errorMessage = document.querySelector('.error-message');
+
+function emailValidation() {
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(String(email.value));
+}
+
+function emailLowerCheck() {
+  const arr = email.value.split('');
+  let result = true;
+  arr.forEach((x) => {
+    // eslint-disable-next-line max-len
```

Comment:
- [ ] Please try to not disable Linter rules, these are in place to make sure that other developers can read easily our code and follow along. The following is the correct way to solve this issue:

![image](https://user-images.githubusercontent.com/79114233/141149263-7961c226-9c6c-45cd-b0e9-22c827c868be.png)

Reply:
Thanks for your feedback. Appreciate it!